From 7619c56ab4ff89a7c491b7cd0454e4a43bf385d4 Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Fri, 22 Jul 2016 09:29:31 +0800
Subject: [PATCH] phy: rockchip-usb: remove the support of rk3366 and rk3399

For rk3366 and rk3399, the usb-phy IP block is different than
previous chips, so we should use a new driver to handle them.

Change-Id: I44ebe883664e9fb2fce8707bab2d9159c546e51f
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 drivers/phy/phy-rockchip-usb.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/phy/phy-rockchip-usb.c b/drivers/phy/phy-rockchip-usb.c
index 28119edc0599..39f419d6d484 100644
--- a/drivers/phy/phy-rockchip-usb.c
+++ b/drivers/phy/phy-rockchip-usb.c
@@ -390,8 +390,6 @@ static const struct of_device_id rockchip_usb_phy_dt_ids[] = {
 	{ .compatible = "rockchip,rk3066a-usb-phy", .data = &rk3066a_pdata },
 	{ .compatible = "rockchip,rk3188-usb-phy", .data = &rk3188_pdata },
 	{ .compatible = "rockchip,rk3288-usb-phy", .data = &rk3288_pdata },
-	{ .compatible = "rockchip,rk336x-usb-phy", .data = &rk336x_pdata },
-	{ .compatible = "rockchip,rk3399-usb-phy", .data = &rk3399_pdata },
 	{}
 };
 
-- 
2.35.3

